### PR TITLE
add event time to entityChangeEventValue

### PR DESCRIPTION
**Pull request by SrikarMannepalli**
## Description
Added the event time to entityChangeEventValue.

### CODE REVIEW

File: entity-service-change-event-api/src/main/proto/org/hypertrace/entity/change/event/v1/entity_change_event_value.proto [hunk 1]
```diff
@@ -14,6 +14,7 @@ message EntityChangeEventValue {
   }
   optional string user_id = 4;
   optional string user_name = 5;
+  int64 event_time_millis = 6;
```
[the remark this answers is not in the document]
Comment (aaron-steinfeld) [reply entity-service-change-event-api/src/main/proto/org/hypertrace/entity/change/event/v1/entity_change_event_value.proto]: adding a new field is compatible, it doesn't need to be optional (remember, optional in proto just signifies a presence check).

[the remark this answers is not in the document]
Comment (SrikarMannepalli) [reply entity-service-change-event-api/src/main/proto/org/hypertrace/entity/change/event/v1/entity_change_event_value.proto]: Updated it now.

[the remark this answers is not in the document]
Comment (aaron-steinfeld) [reply entity-service-change-event-api/src/main/proto/org/hypertrace/entity/change/event/v1/entity_change_event_value.proto]: (doesn't need to, and also shouldn't be)

[the remark this answers is not in the document]
Comment (SrikarMannepalli) [reply entity-service-change-event-api/src/main/proto/org/hypertrace/entity/change/event/v1/entity_change_event_value.proto:17]: Reverted the change.(Not optional now)

[the remark this answers is not in the document]
Comment (saxenakshitiz) [reply entity-service-change-event-api/src/main/proto/org/hypertrace/entity/change/event/v1/entity_change_event_value.proto:17]: If protos are compatible, then we are good. @SrikarMannepalli you will probably still need to handle packets that may not have event time on consumer side.

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**